src/ec/packages/ec-component-hero-banner/hero-banner.story.js

```diff
@@ -15,105 +16,97 @@ import heroBanner from './ecl-hero-banner.html.twig';
 import notes from './README.md';
 
 const PrepareBanner = data => {
+  getExtraKnobs(data);
+  if (data.link.link.label) {
```

Comment:
This order is valid also for the tabs, ideally i would imagine to see "states" | "mandatory elements" | "optional elements", in this order. There might be situation where this could be tricky, but that's the goal.. ;)
This means that if the first knob you define is withing a group, this group will also be the first tab on the panel.